packages/react-native-legal/plugin/src/ios/withIosLegal.ts

```diff
@@ -2,7 +2,7 @@ import path from 'path';
 
 import { type ConfigPlugin, withXcodeProject } from 'expo/config-plugins';
 
-import { generateLicensePlistNPMOutput, scanDependencies } from '../../../plugin-utils/build/common';
+import { generateLicensePlistNPMOutput, scanDependencies } from '../../../../shared/build/index';
```

Comment:
should be imported from `@callstack/react-native-legal-shared` the same like in bare plugin

Reply:
I'd have to take a look at this again, but I think it might've been causing issues, thus it's hardcoded in here. I'll check and come back with the results on Wednesday.